Application/src/Components/MyPoster.js

```diff
@@ -52,6 +54,18 @@ function MyPoster(props) {
               alt={`Poster number ${props.Id}`}
             />
           </div>
+          <div className='PosterPopUpMenuIconContainer'>
+            <DeleteForeverIcon
+              className='deleteIcon'
+              fontSize='large'
+              onClick={() => console.log('Delete pressed')}
+            />
+            <EditIcon
+              className='editIcon'
+              fontSize='large'
+              onClick={() => console.log('Edit pressed')}
+            />
+          </div>

```

Comment:
![CleanShot 2022-07-19 at 12 24 39@2x](https://user-images.githubusercontent.com/66916746/179812090-896376b3-ac29-4509-982c-ced4508869cc.png)

I don't really like the position of the icons but that's the best I could come up with. Feel free to move them if you want.